cloudera/tests/test_integration.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from datadog_checks.cloudera import ClouderaCheck
+from datadog_checks.cloudera.metrics import METRICS, TIMESERIES_METRICS
+
+from .common import CAN_CONNECT_TAGS, CLUSTER_HEALTH_TAGS
+
+
+@pytest.mark.usefixtures('dd_environment')
+@pytest.mark.integration
+def test_given_bad_url_when_check_runs_then_service_check_critical(
+    aggregator,
+    dd_run_check,
+    cloudera_check,
+    instance,
+):
+    with pytest.raises(Exception):
+        # Given
+        check = cloudera_check(instance)
+        # When
+        dd_run_check(check)
+        # Then
+        aggregator.assert_service_check('cloudera.can_connect', ClouderaCheck.CRITICAL)
+
+
+@pytest.mark.usefixtures('dd_environment')
+@pytest.mark.integration
+def test_given_api_v48_endpoint_when_check_runs_then_service_check_ok_and_metrics(
+    aggregator,
+    dd_run_check,
+    cloudera_check,
+    instance,
+):
+    # Given
+    check = cloudera_check(instance)
+    # When
+    dd_run_check(check)
+    # Then
```

Comment:
Totally cool if you find these comments useful, but fyi they are not (and never will be) required https://github.com/DataDog/integrations-core/pull/13260#discussion_r1036265478

Reply:
Yup sounds good, some of these tests were written by Jose so I kept all of them in his style.